cli/popper/commands/cmd_run.py

```diff
@@ -50,11 +50,18 @@
     is_flag=True,
 )
 @click.option(
+    '-e',
     '--engine',
-    help='Specify runtime for executing the workflow [default: docker].',
+    help='Specify runtime for executing the workflow.',
     type=click.Choice(['docker', 'singularity', 'vagrant']),
-    required=False,
-    default='docker'
+    required=False
+)
+@click.option(
+    '-r',
+    '--resource-manager',
+    help='Specify resource manager for executing the workflow.',
+    type=click.Choice(['slurm']),
+    required=False
```

Comment:
this is missing the `host` option. And `host` can be the default and we can remove the `required=False`

Reply:
Same as above

Reply:
same here, it is fine. Nevermind

Reply:
but we do have to add `host` as a choice

Reply:
Yeah i will be removing the `required=False` and add the host option.

Reply:
we can leave `required=False`, my first comment didn't take into account what we discussed on slack regarding config precedence (CLI flag winning over config file)